pkg/controller/migplan/validation.go

```diff
@@ -581,6 +592,83 @@ func (r ReconcileMigPlan) validateConflict(plan *migapi.MigPlan) error {
 	return nil
 }
 
+// Validiate Namespace Selection
+func (r ReconcileMigPlan) validateNamespaceSelection(plan *migapi.MigPlan) error {
+	invalidNamespaces := make([]string, 0)
```

Comment:
Seems like `unauthorized` would be clearer.  I don't think it's a question of valid/invalid.